veniq/dataset_collection/augmentation.py

```diff
@@ -136,55 +122,125 @@ def _is_match_to_the_conditions(
         return False
 
 
-def determine_type(method_node: ASTNode):
+def check_whether_method_has_return_type(
+        method_decl: AST,
+        var_decls: typing.Set[str]) -> InlineTypesAlgorithms:
     """
-
-    :param method_node: Method declaration
-    :return: int - type
+    Run function to check whether Method declaration can be inlined
```

Comment:
Does this function checks if the method has return type or is it can be inlined?